package.json

```diff
@@ -1,31 +1,16 @@
 {
-  "name": "zarrita",
-  "version": "0.1.4",
-  "description": "A minimal, exploratory implementation of the Zarr version 3.0 core protocol.",
-  "main": "zarrita.js",
+  "name": "zarrita-dev",
```

Comment:
This `package.json` serves to run tests and lint `src`, which contains the contents of the zarrita package.